src/pages/bubblegum-v2/update-cnfts.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Updating Compressed NFTs
+metaTitle: Updating Compressed NFTs | Bubblegum V2
+description: Learn how to update compressed NFTs on Bubblegum.
+---
+
+The **updateMetadataV2** instruction can be used to modify the metadata of a Compressed NFT. The Merkle root is updated to reflect the propagated hash of the data, and RPC providers who conform to the [Metaplex DAS API](https://github.com/metaplex-foundation/digital-asset-standard-api) will update their index of the cNFTs.
+
+The metadata can be updated by one of two authorities, depending on if the compressed NFT is a verified item in a collection.
+
+## Update Authority
+
+There are two possible update authorities for a cNFT which are either the tree owner or the collection authority if the cNFT belongs to a collection.
+
+### Collection Authority
+
+If your cNFT belongs to a collection then the update authority of that cNFT will be the authority of the collection. When updating the cNFT you will need to pass in a `coreCollection` arg to the update function.
```

Comment:
Perhaps should call out that this is either collection update authority _or a delegate_?